### DISCUSSION TITLE
Sponsoring "Moonshoot by Techcabal"

### DISCUSSION THREAD

**Discussion by oiclid** [akash-network/community#664] [General]
## What
[Moonshot by Techcabal](https://moonshot.techcabal.com/) is a tech conference by one of Africa's foremost Tech publications.

## When 

The whole event would take place over 5 days (from October 8 -11, 2024), with 2 days  (October 9 & 10) being conference days, and October 8, 11, & 12 are non-conference days during which there'll be mixers plus other events open to selected attendees (investors, founders, and other African Tech insiders). 

<img width="1512" alt="events" src="https://github.com/user-attachments/assets/d6589e29-2f0a-4ca6-8b06-6900444b0c16">

## How

I would recommend going for the Silver Sponsorship. (It can be found on page 29 of the NGN brochure. It is not available on the USD brochure) at NGN7.5 Million (~$5,000), I would consider it the sweet spot for exposure, representation, and marketing.
<img width="1124" alt="Screenshot 2024-08-20 at 14 35 26" src="https://github.com/user-attachments/assets/ae4d636e-c4be-432a-8bb4-f704086181ea">

[MOONSHOT by TechCabal - 2024 Proposal - NGN 1.pdf](https://github.com/user-attachments/files/16676201/MOONSHOT.by.TechCabal.-.2024.Proposal.-.NGN.1.pdf)

[MOONSHOT by TechCabal - 2024 Proposal - USD.pdf](https://github.com/user-attachments/files/16676200/MOONSHOT.by.TechCabal.-.2024.Proposal.-.USD.pdf)

## Why

Like the brochure says:
- Sponsorship announcement on TechCabal’s website and social media
- Branding on event website
- Pre-recorded 3 minute keynote session produced by TechCabal - played during the
conference and uploaded on TechCabal’s Youtube
- Branding on all social media, email marketing and PR
- 2 Full access delegate and 1 speaker pass.

Plus: 

- Access to founders, investors, insiders, etc. during off-conference events.

## Who

@oiclid and @popeblack2. Both are Akash Insiders.

## Deadline 

If we were to sponsor the event, we would need to make a decision and pay by September 6, 2024.

## How Much

(I have used the figures from the [events budget](https://docs.google.com/spreadsheets/d/1DBXUhNsf8qxIYD-NtuLeGXhmQ-WTOdcCfdyvXgaGGV0/edit?gid=580542914#gid=580542914) to make sure nothing goes over the budget)

Sponsorship: $5000
Return Flight Tickets (x2) = $200 x 2 = $400
Accommodation = $100 x5 x 2 = $1000
Transportation = $50 x 5 x 2 = $500
Feeding = $100x 5 x 2 = $1000

Total = $7900

Thanks. 
I look forward to your comments.

**Comment by ghost** [comment]
I read through the proposal, and I would support, I guess you could add links to TechCabal channels like Twitter and Youtube, and an estimate projection on how much exposure Akash could really get 

I just mention this because you focused more on the exposure and marketing aspect in your proposal, if you think there are other things that Akash could benefit from this event feel free to add them

**Comment by oiclid** [comment]
@instafinanzas Thanks. I have already told the TechCabal team that we will not be sponsoring it. Tha said, I still intend to attend, and would be able to give the team an "on-the-ground" report on what it is like and if we should include it as a potential event in the future.
